docs/UserGuide.md

```diff
@@ -255,12 +271,12 @@ Example of usage:
 ```lang-none
 >> expiring
 These are the food expiring in the next week:
-1. Food name: chicken wings, category: MEAT, expiry: 21-03-2021, stored in: MIDDLE_SHELF
+1. Food name: chicken wings, category: READY_TO_EAT, expiry: 17-04-2021, stored in: UPPER_SHELF, quantity: 500
 ```
 
 ### List categories with food running low: `runninglow`
 
-Displays a list of food categories which total quantity is below a specified minimum limit.
+Displays a list of food categories which total quantity is below a specified minimum limit. This command also displays the existing quantities along with its limits for each category. Note that these limits can be modified with the [`setlimit`](#modify-the-minimum-quantity-limits-setlimit) command.
```

Comment:
Should we inform the user that if all of the minimum quantity if set to 0 would disable the running low command ?

Reply:
rejected, error message on your side will change